### PR TITLE
Add Drupal gitops sync job - Added drupal phpqa template to jobs

### PR DESCRIPTION
**Pull request by andypanix**

### CODE REVIEW

File: templates/jobs/gitops-sync-to-infrastructure-repo.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    - export INFRASTRUCTURE_REPO_DIR="${CI_PROJECT_DIR}/infrastructure/"
+    - export INFRASTRUCTURE_REPO_GIT_URL="https://${INFRASTRUCTURE_REPO_ACCESS_TOKEN}@${INFRASTRUCTURE_REPO_BASE_URL}"
+    # Init Helm 3.
+    - helm3 repo add "stable" "https://charts.helm.sh/stable"
+    - helm3 repo add "sparkfabrik" "https://storage.googleapis.com/spark-helm-charts"
+    - helm3 repo update
+    # Clone environment repo.
+    - mkdir -p ${INFRASTRUCTURE_REPO_DIR}
+    - cd ${INFRASTRUCTURE_REPO_DIR}
+    - git config --global user.email "gitlab-bot@sparkfabrik.com"
```
Comment (paolomainardi) [inline templates/jobs/gitops-sync-to-infrastructure-repo.yml:50]: Change it to `gitlab-gitops-bot@sparkfabrik.cloud`

Comment (andypanix) [reply templates/jobs/gitops-sync-to-infrastructure-repo.yml]: Corretto 👍

File: templates/jobs/gitops-sync-to-infrastructure-repo.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    - helm3 template ${HELM_DEPLOY_NAME} sparkfabrik/drupal
+      --version 3.3.0
+      -f ${INFRASTRUCTURE_REPO_DIR}/values.yaml
+      --output-dir ${INFRASTRUCTURE_REPO_DIR}/generated-manifests
+      --render-subchart-notes
+      --include-crds
+      --wait
+    # Commit updated manifests to the repo.
+    - git add .
+    # @TODO: find a better solution than allow-empty flag to prevent push to repo when we do not have changes.
```
Comment (paolomainardi) [inline templates/jobs/gitops-sync-to-infrastructure-repo.yml:67]: i think that you can just use a git diff approach to test this: https://git-scm.com/docs/git-diff#Documentation/git-diff.txt---exit-code

Comment (andypanix) [reply templates/jobs/gitops-sync-to-infrastructure-repo.yml]: @paolomainardi Fixed with git diff

**Pull request merged**